docs-elastic/index.asciidoc

```diff
@@ -6,4 +6,6 @@ The {n} allows you to provision Elastic Cloud deployments on any Elastic Cloud p
 
 The provider lets you manage Elastic Cloud deployments as code, and introduce DevOps-driven methodologies to manage and deploy the Elastic Stack and solutions.
 
+IMPORTANT: The provider is not subject to the support service level agreement (SLA.) You can use the [Elastic Cloud APIs](https://www.elastic.co/guide/en/cloud/current/ec-restful-api.html) or [Elasticsearch Service Console](https://www.elastic.co/guide/en/cloud/current/ec-configure-settings.html) directly to perform changes as a workaround.
```

Comment:
> directly to perform changes as a workaround

For many people this is unlikely to be true. Terraform is often setup as the _only_ way changes can get into production. Intentionally introducing drift from the desired state (as defined in the Terraform resources) can also be a pretty big risk for managed resources, reconciling that drift can easily create outages. I don't think we should casually direct people down this path. 

What's the reason this client is being carved out of the SLA and not Cloud UI?

Reply:
Thanks for the feedback! This PR is the outcome from an internal discussion thread.  I have looped you in so we can continue to refine the arrangement.